controlplane/kubeadm/controllers/controller_test.go

```diff
@@ -60,7 +60,7 @@ var _ = Describe("KubeadmControlPlaneReconciler", func() {
 
 	Context("Reconcile a KubeadmControlPlane", func() {
 		It("should return error if owner cluster is missing", func() {
-			clusterName, clusterNamespace := "foo", "default"
+			clusterName, clusterNamespace := "foo-1", "default"
```

Comment:
It seems like this should be using a generated namespace rather than "default", possibly one created in `BeforeEach` and deleted in `AfterEach`